apps/api-extractor/src/api/IConfigFile.ts

```diff
@@ -405,4 +406,15 @@ export interface IConfigFile {
    * to prevent spurious diffs in output files tracked for tests.
    */
   testMode?: boolean;
+
+  /**
+   * Specifies how API Extractor sorts members of an enum when generating api.json, api.md and rollup.d.ts.
```

Comment:
This config doesn't affect `.api.md` or `.d.ts` rollups, so this comment should be updated. Maybe:

> Specifies how API Extractor orders enum members in the .api.json file.